### DISCUSSION TITLE
Cannot disable directory auto folding

### DISCUSSION THREAD

**Discussion by mjdNjhNJ** [zed-industries/zed#16655] [Feature Requests]
I have set `"auto_fold_dirs": false`, as it's the default value as well. However, the project panel is still behaving as if it been se to `true`. Is it bugged or am I missing the point of the config?

<img width="325" alt="image" src="https://github.com/user-attachments/assets/7858a682-a834-4d0c-b1ce-1dcd0e3c0bb3">

I would expect it to not be folded.

**Comment by SomeoneToIgnore** [comment]
There has to be some confusion which I have fixed in the docs, as `auto_fold_dirs` is `true` by default for some while.

I got it disabled both in latest Preview and Stable:

https://github.com/user-attachments/assets/b761d908-8a6f-44be-9b68-f3971e1caae2

and the only related issue I've noticed is that the config file has to be saved twice in order for the setting change to apply? 
Not sure what's wrong with that yet, but disabling seems to be possible at least.

**Reply by mjdNjhNJ** [reply]
I have missed that I need to set this inside the `project_panel` object. Thanks for updating the docs, now it works as expected. However, I can reproduce the same issue with a need to save the setting file two times for changes to apply.

**Reply by SomeoneToIgnore** [reply]
Which seems to be https://github.com/zed-industries/zed/issues/16351 

Very odd that it's the first time I stumble on this, but maybe it is related to certain properties only.
Thank you for confirming.
